articles/postgresql/flexible-server/concepts-elastic-clusters-limitations.yml

```diff
@@ -25,74 +25,79 @@ sections:
+
+      - question: Why isn't TimescaleDB available with elastic clusters?
+        answer: Elastic clusters make use of key low-level system hooks which conflict with the TimescaleDB extension.  Fortunately, elastic clusters provide similar time-series capabilities.  You can learn more about timeseries data in the [Citus documentation](https://docs.citusdata.com/en/stable/use_cases/timeseries.html).
 
   - name: Migrations
     questions:
       - question: How can I migrate to/from elastic clusters?
-        answer: Currently, migrations to/from elastic clusters can be done with pg_dump, pg_restore, and pgcopydb. Any other tool working with standard PostgreSQL should work.
+        answer: Migrations to/from elastic clusters can be done with pg_dump, pg_restore, and pgcopydb. Any other tool working with standard PostgreSQL should work.
+
+      - question: Can I convert my existing Public Preview instance to General Availability (GA) elastic clusters?
+        answer: If you have created any elastic clusters during the Public Preview phase, you are required to re-create them once the feature becomes Generally Available.  If this raises concerns, or if you need help determining whether your server meets the necessary requirements, please contact us by going to the Azure portal, selecting your Azure Postgres resource, and following the instructions in the banner.
+
   - name: Networking
     questions:
       - question: Can I use PgBouncer for connection pooling with elastic clusters?
-        answer: Yes, you can use PgBouncer with elastic clusters. Port 6432 should be used for schema and node management operations, while port 8432 is load-balanced to PgBouncer instances running across all nodes in the cluster.
+        answer: Yes, PgBouncer is supported with elastic clusters. Port 6432 should be used for schema and node management operations, while port 8432 is load-balanced to PgBouncer instances running across all nodes in the cluster.
 
-      - question: Can I use virtual network (VNet) with elastic clusters?
-        answer: Currently, virtual network isn't supported during the preview.
+      - question: Can I use virtual network (VNet) injection with elastic clusters?
+        answer: Currently, virtual network injection isn't supported during the preview.
 
       - question: Are Private Domain Name System (DNS) Zones supported with elastic clusters?
         answer: Currently, Private DNS Zones aren't supported during the preview.
 
   - name: Storage
     questions:
       - question: Can I use customer-managed keys (CMK) for storage encryption?
-        answer: Currently, customer-managed keys aren't supported during the preview.
+        answer: Customer-managed keys are not currently supported during the preview.
```

Comment:
We should add this will be supported before Elastic clusters are general available.